HelpSource/Classes/QuartzComposerView.schelp

```diff
@@ -193,7 +212,7 @@ b = Button(w, Rect(0, 0, 150, 20))
     .states_([["pick another QC file"]])
     .action_({ File.openDialog("", { |path| m.path_(path) }) });
 m = QuartzComposerView(w, Rect(0,20,400, 260));
-m.path = Platform.helpDir +/+ "QC/ImageFilter.qtz";
+m.path = Platform.userAppSupportDir +/+ "Help/QC/ImageFilter.qtz"; // file missing
```

Comment:
if the file is gone, just delete the example, don't leave a non-working example in the documentation